perllib/FixMyStreet/App/Controller/Dashboard.pm

```diff
@@ -289,6 +289,29 @@ sub generate_grouped_data : Private {
             my $bm = $map{$b} // $map{$state_map->{$b}};
             $am <=> $bm;
         } @rows;
+    } elsif ($group_by eq 'category+state' || $group_by eq 'category') {
+        @rows = ();
+        my %category_to_group;
+        for my $group (@{$c->stash->{category_groups}}) {
+            for my $categories ($group->{categories}) {
+                for my $category (@$categories) {
+                    push @rows, $category->category;
+                    if (!$category_to_group{$category->category}) {
+                        $category_to_group{$category->category} = $group->{name};
+                    } else {
+                        $category_to_group{$category->category} = 'Multiple';
+                    }
+                }
+            }
+        };
+
+        my @multiple_groups = grep { $category_to_group{$_} eq 'Multiple'} @rows;
```

Comment:
And here, could use List::MoreUtils' part to have `my ($non_multiple, $multiple) = part { $category_to_group{$_} eq 'Multiple' } @rows;`

So this section becomes something like:
```
my ($non_multiple, $multiple) = part { $category_to_group{$_} eq 'Multiple' } @rows;
my @multiple = sort(uniq(@$multiple));
@rows = (@$non_multiple, @multiple);
```